services-js/311/server/server.ts

```diff
@@ -103,16 +105,21 @@ export default async function startServer({ rollbar }: { rollbar: Rollbar }) {
     process.env.ELASTICSEARCH_INDEX
   );
 
-  const salesforce = process.env.SALESFORCE_OAUTH_URL
-    ? new Salesforce(
-        process.env.SALESFORCE_OAUTH_URL,
-        process.env.SALESFORCE_CONSUMER_KEY,
-        process.env.SALESFORCE_CONSUMER_SECRET,
-        process.env.SALESFORCE_API_USERNAME,
-        process.env.SALESFORCE_API_PASSWORD,
-        process.env.SALESFORCE_API_SECURITY_TOKEN
-      )
-    : null;
+  // If we're in maintenance mode we don’t try to connect to Salesforce because
```

Comment:
I can imagine this comment saving someone from a lot of unnecessary anxiety in the future! 😆

Reply:
The best kind of comment!